src/cpp/r/session/RClientState.cpp

```diff
@@ -232,24 +240,24 @@ void ClientState::putProjectPersistent(const std::string& scope,
                                        const json::Value& value)
 {
    json::Object stateContainer;
-   putState(scope, json::Member(name, value), &stateContainer);
+   putState(scope, name, value, &stateContainer);
    putProjectPersistent(stateContainer);
 }
 
 json::Value ClientState::getProjectPersistent(std::string scope,
                                               std::string name)
 {
-   json::Object::iterator i = projectPersistentState_.find(scope);
+   json::Object::Iterator i = projectPersistentState_.find(scope);
    if (i == projectPersistentState_.end())
    {
       return json::Value();
    }
    else
    {
-      if (!json::isType<core::json::Object>((*i).value()))
+      if (!json::isType<core::json::Object>((*i).getValue()))
          return json::Value();
-      json::Object scopeObject = (*i).value().get_obj();
-      return scopeObject[name].clone();
+      json::Object scopeObject = (*i).getValue().getObject();
+      return json::Value(scopeObject[name]);
```

Comment:
Curious why not just clone here? Did we remove that?

Reply:
I did, and then I un-removed it and I missed this one when going through old changes to add back clone calls. Nice catch!